src/openforms/scss/components/admin/_datetime.scss

```diff
@@ -0,0 +1,10 @@
+.datetime-input {
+  display: inline-flex;
+  gap: 3px;
+  padding-right: 2px;
+
+  i {
+    margin-top: 2px;
```

Comment:
perhaps applying `align-items: center` on the parent works better to vertically center things?

Reply:
Yes, this is what I tried first but it isn't shown correctly

![image](https://github.com/open-formulieren/open-forms/assets/63058071/e284125f-eee7-4198-b52c-34ffc6f05720)